Formula/mysql.rb

```diff
@@ -86,6 +85,7 @@ def install
       -DWITH_UNIT_TESTS=OFF
       -DENABLED_LOCAL_INFILE=1
       -DWITH_INNODB_MEMCACHED=ON
+      -DWITH_ZLIB=bundled
```

Comment:
Let's not do that. Instead, remove
```ruby
uses_from_macos "zlib"
```
and add
```ruby
depends_on "zlib" # Zlib 1.2.12+
```

Reply:
Thanks for the suggestion.